Frontend/src/App.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { useState } from 'react'
+
+import Homepage from './pages/homepage/Homepage'
+import './App.css'
+
+function App() {
+  const [count, setCount] = useState(0)
+
+  return (
+    <div>
```

Comment:
no need for this `div` you can do:
```suggestion
    <>
        <Homepage/>
    </>
```